### PR TITLE
8328004: Minor updates to TrayIcon test DisposeInActionEventTest.java

### PR DESCRIPTION
**Pull request by prrace**
This was recently converted from a manual Applet to a PassFailJFrame main program
But I see a few things that could be tweaked
 - Clarify instructions
 -  Set path to PassFailJFrame
 - reduce timeout to default
 - eliminate redundant folder

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328004](https://bugs.openjdk.org/browse/JDK-8328004): Minor updates to TrayIcon test DisposeInActionEventTest.java (**Bug** - P4)

### Reviewers
 * [Alexander Zvegintsev](https://openjdk.org/census#azvegint) (@azvegint - **Reviewer**) ⚠️ Review applies to [800c3c43](https://git.openjdk.org/jdk/pull/18246/files/800c3c4382235391e1bbb18fda9217f71f89121d)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18246/head:pull/18246` \
`$ git checkout pull/18246`

Update a local copy of the PR: \
`$ git checkout pull/18246` \
`$ git pull https://git.openjdk.org/jdk.git pull/18246/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18246`

View PR using the GUI difftool: \
`$ git pr show -t 18246`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18246.diff">https://git.openjdk.org/jdk/pull/18246.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18246#issuecomment-1992335568)

### CODE REVIEW

File: test/jdk/java/awt/TrayIcon/DisposeInActionEventTest.java [hunk 1]
```diff
@@ -57,24 +57,26 @@ public static void main(String[] args) throws Exception {
         String clickInstruction =
                 (Platform.isOSX()) ? "Right-click" : "Double click (left mouse button)";
```
Comment (azvegint) [inline test/jdk/java/awt/TrayIcon/DisposeInActionEventTest.java:58]: On Linux, it triggers with a single click(Tested on Ubuntu 23.10)

**Review by TejeshR13 — COMMENTED** [review]
`textArea` can be moved inside `showFrameAndIcon()`.

File: test/jdk/java/awt/TrayIcon/DisposeInActionEventTest.java (hunk 1 above)
Comment (prrace) [reply test/jdk/java/awt/TrayIcon/DisposeInActionEventTest.java:58]: Manual tests that require a tester's interpretation and that are specific to the way a platform expects user interaction can be  subjective but the idea is that they are objective. I already didn't like the clickInstruction logic, now I  like it even less.

**Review by azvegint — APPROVED** [review]

**Pull request closed without merging**